Mathlib/Data/Finset/Sum.lean

```diff
@@ -35,11 +35,9 @@ def disjSum : Finset (α ⊕ β) :=
 theorem val_disjSum : (s.disjSum t).1 = s.1.disjSum t.1 :=
   rfl
 
-@[simp]
 theorem empty_disjSum : (∅ : Finset α).disjSum t = t.map Embedding.inr :=
```

Comment:
What's the motivation for dropping `simp` here?

Reply:
This makes
```
lemma toRight_eq_empty : u.toRight = ∅ ↔ u ⊆ univ.disjSum ∅ := by simp [subset_disjSum]
```
work. In general, I think `empty_disjSum` is a bad idea because it replaces "specifc map along a function with known inverse" (`(∅ : Finset α).disjSum ·)` with `Finset.toRight`) with "generic map with no known inverse" (`(·.map Embedding.inr)`)

Reply:
`Finset.toRight` is the inverse of the right hand side too though - I'm not quite sure what you mean. I view this lemma as saying 0 + t = t, and so should obviously be a simp lemma. `(0 + ·)` is a specific map, and replacing it with generic `(·)` is a good thing!

Reply:
Okay, what about now?